src/tmx_json.c

```diff
@@ -131,8 +131,9 @@ static int pjson_layer(json_t *lay_el, tmx_layer **lay_headaddr, const char *fil
 	lay->next = *lay_headaddr;
 	*lay_headaddr = lay;
 
-	if (json_unpack_ex(lay_el, &err, 0, "{s:b, s:F, s:s, s:s}",
+	if (json_unpack_ex(lay_el, &err, 0, "{s:b, s:F, s:i, s:i, s:s, s:s}",
 	                   "visible", &(lay->visible), "opacity", &(lay->opacity),
+					   "x",       &(lay->x_offset), "y",      &(lay->y_offset),

```

Comment:
Please use tabs for indenting and spaces for alignment, so then the source looks nice whatever the tab length you use.